graphics/DiffPDF/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           qmake 1.0
+
+name                DiffPDF
+version             2.1.3
+categories          graphics
+platforms           darwin
+license             GPL-2+
+maintainers         {gmail.com:davide.liessi @dliessi} openmaintainer
+description         Compare two PDF files textually or visually
+long_description    ${name} can compare two PDF files. \
+                    It offers three comparison modes: \
+                    Words, Characters, and Appearance.
+homepage            http://www.qtrac.eu/diffpdf-foss.html
+master_sites        http://www.qtrac.eu/
+
+distname            [string tolower ${name}-${version}]
+
+checksums           sha1    663ecb7666a4b7d6fbd1a37327fd1f895fae69b8 \
+                    rmd160  12a5f6921b8a53d98d9786af25e408e27611f56f \
+                    sha256  c6142ee038a78108397f45b0c456dca7a4fe1d75250f21a514a134101d322433 \
+                    size    91856
+
+depends_lib-append  port:poppler-qt4-mac
+configure.env-append PKG_CONFIG_PATH=${prefix}/libexec/poppler-qt4-mac/lib/pkgconfig
+build.env-append PKG_CONFIG_PATH=${prefix}/libexec/poppler-qt4-mac/lib/pkgconfig
+
+patchfiles          patch-include-poppler-qt4.diff
+
+post-patch {
+    reinplace s|@@PREFIX@@|${prefix}| ${worksrcpath}/[string tolower ${name}].pro
+    reinplace s|@@ICON@@|${filespath}/icon.icns| ${worksrcpath}/[string tolower ${name}].pro
+}
+
+pre-configure {
+    system -W ${worksrcpath} "${qt_lrelease_cmd} [string tolower ${name}].pro"
+}
+
+post-build {
+    move ${worksrcpath}/[string tolower ${name}].app ${worksrcpath}/${name}.app
+    system -W ${worksrcpath} "${qt_bins_dir}/macdeployqt ${name}.app"
+    foreach framework { QtDeclarative.framework \
+                        QtNetwork.framework \
+                        QtScript.framework \
+                        QtSql.framework \
+                        QtSvg.framework \
+                        QtXmlPatterns.framework } {
+        delete ${worksrcpath}/${name}.app/Contents/Frameworks/${framework}
```

Comment:
Would be better to modify the build system so that it does not copy these frameworks in the first place, to save time. If the build system doesn't offer an option for that, maybe ask the developers to add such an option.

Reply:
I added that part because the build instructions for macOS said so, but it turns out macdeployqt is smart enough in this case and those frameworks were not copied into the application bundle in the first place.